tests/test_example_assert_dashboard_page.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from selenium import webdriver
+
+from pages.DashboardPage import DashboardPage
+from pages.LoginHrmPage import LoginHrmPage
+
+
+class TestExampleAssertDashboardPage:
+
+    def test_assert_dashboard_page(self):
+        driver = webdriver.Chrome()
+        driver.get("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login")
+        time.sleep(2)
+        login_hrm_page = LoginHrmPage(driver)
+        login_hrm_page.type_username("Admin")
+        login_hrm_page.type_password("admin123")
+        login_hrm_page.click_button_login()
+        time.sleep(5)
+
+        dashboard_page = DashboardPage(driver)
+        assert dashboard_page.get_title_time_at_work() == "Time at Work"
```

Comment:
Please add all the error messages @andyloose

Reply:
fixed all comments